### DISCUSSION TITLE
Bubble map implementation

### DISCUSSION THREAD

**Discussion by NeldoAgustin** [plouc/nivo#1999] [Q&A]
Hello everyone, is there a way to reproduce [D3's bubblemap](https://d3-graph-gallery.com/graph/bubblemap_circleFeatures.html)?
I was thinking on using [ChoroplethCanvas](https://nivo.rocks/choropleth/canvas/) but I need to add bubbles (with tooltip) on top of it. I've seen there is the `layers` prop but the docs don't show an example of its usage or if it's gonna be enough to achieve what I want.

So I'm asking if any1 has done this before with nivo or if I should use D3 for this specific case. Thanks in advance :D

![image](https://user-images.githubusercontent.com/101656192/168128400-f53e77d6-87da-4055-a35f-787501431af2.png)

**Comment by plouc** [comment]
In theory, this should be feasible using the `GeoMap` component from `@nivo/geo` and a custom layer, but I don't have any example available, this would require digging into the code, which isn't necessarily easy.
